### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Mention the Issue Summary Comment for contributors to look for when trying to find an issue to contribute to

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To help you be successful we've put together a few simple rules to follow when y
...
   In all these cases we had to say `no` to those PRs and close them. That, obviously, is not a great outcome for us. And it's especially bad for the contributor, as they've spent a lot of effort preparing the change.
-  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted).
+  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, look for a comment from our team, which follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as procvides references and hits about how to approach the problem at hand.
```
Comment (danmoseley) [inline CONTRIBUTING.md:45]:
```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.
```

Comment (danmoseley) [reply CONTRIBUTING.md:45]: I assume we won't be able to annotate all the existing help wanted (?)

**Review by danmoseley — APPROVED** [review]

**Comment by danmoseley** [comment]
Random idea - we could link canned queries for help wanted issues in each area. (??)

"If you are interested in working on Blazor, look at this list..." type thing

File: CONTRIBUTING.md (hunk 1 above)
Comment (Kahbazi) [inline CONTRIBUTING.md:45]:
```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, look for a comment from our team, which follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is meant to make it much simpler for you to both understand what the problem is as well as procvides references and hits about how to approach the problem at hand.
```

Comment (captainsafia) [reply CONTRIBUTING.md:45]: > I assume we won't be able to annotate all the existing help wanted (?)

Yep, there's a few issues that already have summaries that need to be updated to follow the format but we'll need to make concerted effort to update the rest.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ To help you be successful we've put together a few simple rules to follow when y
...
   In all these cases we had to say `no` to those PRs and close them. That, obviously, is not a great outcome for us. And it's especially bad for the contributor, as they've spent a lot of effort preparing the change.
-  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted).
+  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.
```
Comment (mkArtakMSFT) [inline CONTRIBUTING.md:45]:
```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is meant to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.
```

**Comment by mkArtakMSFT** [comment]
Thanks @danmoseley and @Kahbazi for your reviews.

**Comment by mkArtakMSFT** [comment]
> Random idea - we could link canned queries for help wanted issues in each area. (??)
> 
> "If you are interested in working on Blazor, look at this list..." type thing

Maybe. Our `area-` labels are quite intuitive. Let's wait for more feedback on this and see if this is really something people find hard without.

**Comment by danmoseley** [comment]
Note we have https://github.com/dotnet/aspnetcore/contribute

**Comment by mkArtakMSFT** [comment]
> Note we have https://github.com/dotnet/aspnetcore/contribute

Can we tweak that view (filter it to only include help-wanted issues on that list) ?

**Comment by mkArtakMSFT** [comment]
@danmoseley I'm going to merge this and we can have the conversation separately.

**Comment by danmoseley** [comment]
> Can we tweak that view (filter it to only include help-wanted issues on that list) ?

don't know -- probably not? I think it's Github generated -- you can do it for any repo by editing the URL. Not sure how people find it.

**Pull request merged**